src/Lecture2.hs

```diff
@@ -97,7 +109,10 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+
+dropSpaces :: [Char] -> [Char]
+dropSpaces string = dropWhile space (reverse (dropWhile space (reverse string)))
```

Comment:
I'm afraid, this implementation hangs on infinite lists because `reverse` doesn't work for infinite lists ♾️ 

You can fix this by using the `takeWhile` function in addition to `dropWhile`